helm/cdash/templates/worker/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: "{{ .Release.Name }}-worker"
+  labels:
+    app: "{{ .Release.Name }}-worker"
+spec:
+  replicas: {{ .Values.cdash.worker.replicas }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: "{{ .Release.Name }}-worker"
+  template:
+    metadata:
+      labels:
+        app: "{{ .Release.Name }}-worker"
+    spec:
+      containers:
+        - name: "{{ .Release.Name }}-worker"
+          image: {{ .Values.cdash.image | quote }}
+          resources:
+            {{ toYaml .Values.cdash.worker.resources | nindent 12 }}
+          command: [ "/bin/bash", "/cdash/docker/docker-entrypoint.sh" ]
+          args: [ "start-worker" ]
+          env:
+          - name: "APP_URL"
+            value: "http://{{ .Release.Name }}-website:8080"
```

Comment:
Do the workers ever need to use the `APP_URL`?  If we require S3/minio, we're guaranteed to have a shared filesystem and shouldn't ever need to use remote workers.  I can't think of any other case where we'd need to be able to contact the primary server.